src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -834,8 +835,9 @@ private async Task<DownloadedBuild> GatherDropForBuildAsync(Build build, string
                 string extraAssetsDirectory = Path.Join(rootOutputDirectory, "extra-assets");
                 Directory.CreateDirectory(extraAssetsDirectory);
 
-                (bool success, ConcurrentBag<DownloadedAsset> downloadedExtraAssets) extraAssetDownloadResult = await DownloadAssetsToDirectories(mustDownloadAssets, build, extraAssetsDirectory, unifiedOutputDirectory);
+                (bool success, bool anyShipping, ConcurrentBag<DownloadedAsset> downloadedExtraAssets) extraAssetDownloadResult = await DownloadAssetsToDirectories(mustDownloadAssets, build, extraAssetsDirectory, unifiedOutputDirectory);
                 extraDownloadedAssets = extraAssetDownloadResult.downloadedExtraAssets;
+                anyShipping |= extraAssetDownloadResult.anyShipping;
```

Comment:
I don't think we care about anyShipping for the extra assets, which is probably how this bool went missing in the first place. I think this can be removed.